### PR TITLE
[No QA] Remove unused network loading key

### PR DESCRIPTION
**Pull request by tgolen**
### Details
This is no longer used since the reconnection spinner was removed from the avatar

### Tests
None

- [ ] Verify that no errors appear in the JS console

### CODE REVIEW

File: src/libs/NetworkConnection.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import _ from 'underscore';
 import NetInfo from '@react-native-community/netinfo';
 import AppStateMonitor from './AppStateMonitor';
-import promiseAllSettled from './promiseAllSettled';
```
Comment (luacmartins) [inline src/libs/NetworkConnection.js:4]: `promiseAllSettled` seems to be unused in the code base now. I think we should remove it as well.

**Comment by tgolen** [comment]
Updated

**Review by luacmartins — APPROVED** [review]
LGTM

**Comment by luacmartins** [comment]
Can we add a link to an issue or comment so we can track where these changes are coming from?

**Pull request merged**